Parser.fs

```diff
@@ -557,22 +565,18 @@ let parseMethod =
                 // ^-                             ... <block>
                 (fun s b -> {Signature = s ; Body = b} )
 
-/// Parses a variable with the given initial keyword.
-let parseVar kw = pstring kw >>. ws
-                  // ^- shared     ...
-                             >>. parseTypedTypedVar
-                             // ^- ... <type> <identifier> ...
-                             .>> pstring ";"
-                             // ^-                         ... ;
+/// Parses a variable set with the given initial keyword and AST type.
+let parseVars kw atype =
+    let parseList = parseParams parseIdentifier .>> wsSemi
+    pstring kw >>. ws >>. pipe2ws parseType parseList (fun t v -> atype (t, v))
```

Comment:
I guess they seemed useful to me, but I take your point re maintenance. Anyway, it's not worth obsessing over.